inc/best-offers/best-offers.inc

```diff
@@ -42,10 +42,10 @@
                 <span class="distance"><i class="fas fa-road right"></i>12,000 KM</span>
               </div>
               <div class="model">
-                <a href="">Used</a>>
-                <a href="">2015</a>>
-                <a href="">Automatic</a>>
-                <a href="">Red</a>>
+                <a href="">Used</a> &bull;
```

Comment:
Remove anything after the "a" tag I see a weird character after the "a" tag "&bull".